pysam/libcbcf.pyx

```diff
@@ -1184,6 +1185,17 @@ cdef inline bcf_sync_end(VariantRecord record):
         bcf_info_set_value(record, b'END', record.ptr.pos + record.ptr.rlen)
 
 
+cdef inline int has_symbolic_allele(VariantRecord record):
+    """Return index of first symbolic allele. 0 if no symbolic alleles."""
+
+    for i in range(1, record.ptr.n_allele):
+        alt = record.ptr.d.allele[i]
```

Comment:
Does `alt` need a cdef?  (This requires checking the generated Cython)

Reply:
Sorry, we're hitting the limits of my Cython here. How do you tell if it needs a `cdef`? I checked the generated C and it looks like Cython added a declaration for `alt`. Is that what you meant?

```
static CYTHON_INLINE int __pyx_f_5pysam_7libcbcf_has_symbolic_allele(struct __pyx_obj_5pysam_7libcbcf_VariantRecord *__pyx_v_record) {
  long __pyx_v_i;
  char *__pyx_v_alt;
  int __pyx_r;
  __Pyx_TraceDeclarations
  __Pyx_RefNannyDeclarations
  uint32_t __pyx_t_1;
  long __pyx_t_2;
  int __pyx_t_3;
  int __pyx_t_4;
  size_t __pyx_t_5;
```